packages/samples/headless-react/src/pages/AtomicReactPage.tsx

```diff
@@ -208,3 +138,72 @@ export function AtomicReactPage() {
     </AtomicSearchInterface>
   );
 }
+
+function MyTemplate(result: Result) {
+  return (
+    <>
```

Comment:
is this like a Fragment?

Reply:
It's a way to return a "single root" for a JSX function, without the need for any extra DOM node.

https://reactjs.org/docs/fragments.html